package.json

```diff
@@ -34,8 +34,11 @@
 		"fix": "npm run format:fix && npm run lint:fix"
 	},
 	"devDependencies": {
+		"@jazzer.js/core": "^1.1.0",
```

Comment:
Are the `jazzer.js` dev dependencies necessary? And if so, could we get rid of the specific version number?

Reply:
Not quite sure how that got in there. Removed it.